pkg/workflows/utils.go

```diff
@@ -72,3 +72,15 @@ func GenerateWorkflowID(owner []byte, name string, workflow []byte, config []byt
 
 	return sha, nil
 }
+
+// NormalizeWorkflowName returns the first 10 bytes of the SHA-256 hash of the input string.
+func NormalizeWorkflowName(name string) [10]byte {
```

Comment:
Can we name this a little more directly? And consider that we are already in the workflows package, so repeating workflow is noisy for the caller:
```suggestion
func NameHashPrefix(name string) [10]byte {
```
🤷